src/notest_stack.py

```diff
@@ -73,21 +71,20 @@ def test_pop_empty():
 
```

Comment:
It's great that you have this test, however it is not testing for the right thing. It should be testing that pop() raises an error when trying to pop from an empty Stack. You can do that with `pytest.raises`:
```
with pytest.raises(IndexError):
    test_stack.pop()
```
There is no explicit assert statement; the pytest.raises function handles the expected exception and raises an assertion error only if the expected error is not raise.